Configuration/Adaptive_Mesh.cfg

```diff
@@ -35,6 +35,7 @@ variable_detach_macro: 'Dock_Probe'     # Here is where you define the macro tha
 variable_display_parameters: True       # Display macro paramters in the console, useful for debugging the SETUP_KAMP_MESHING call, or more verbosity.
 
 gcode:
+    {% set mesh_profile_name = params.PROFILE|default('default') %}
```

Comment:
Any chance we could make it default to a variable we add above and then update the `SETUP_KAMP_MESHING` to set that default?

Reply:
Well, the problem is, the `BED_MESH_CALIBRATE` is quite often used with parameter `PROFILE=` and move profile name definition to the `SETUP_KAMP_MESHING` breaks this "backward" compatibility.
Anyway, yes it is absolutely possible. Give me a moment

Reply:
There is no place to report issues.  I have an issue where multi-material prints (like with the ERCF) don't use the correct bounds as the purge tower and priming lines now intersect with where the KAMP does it's purge line, creating a nozzle collision.  Any chance this pull request does something to fix that?

Reply:
@jwhite How is it related with this pull request?

Reply:
@kmarty Thanks.  There is no way to report an issue or to talk to the author, other than this message thread.